src/main/java/seedu/address/logic/commands/ReturnCommand.java

```diff
@@ -23,7 +23,7 @@ public class ReturnCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the book list of the person identified "
             + "by the index number used in the last person listing. "
             + "If book exists in the person's book list, it will be removed. \n"
-            + "Parameters: INDEX (must be a positive integer) \n"
+            + "Parameters: INDEX (must be a positive integer) "
```

Comment:
This change was deemed necessary by our group as without removing this line break at this line, users may misinterpret the command syntax to be used for the `return` command, resulting in a functional flaw.